helix-view/Cargo.toml

```diff
@@ -50,6 +50,7 @@ toml = "0.8"
 log = "~0.4"
 
 parking_lot = "0.12.2"
+thiserror = "1.0"
```

Comment:
Could you set this up as a workspace dependency? `slotmap` is a good example of this - in this `Cargo.toml` and in `helix-lsp/Cargo.toml` and `helix-dap/Cargo.toml` we should set `thiserror.workspace = true` and then in the root `Cargo.toml` we can set a single version `thiserror = "1.0"`

Reply:
Sounds great! Just committed this change also.